Definitions/Chord

```diff
@@ -1,18 +1,27 @@
 class Chord:
-    def __init__(self, definition):
+    def __init__(self, definition, inversion):
         self.symbol = definition.symbol
         self.degrees = definition.degrees
         self.next = definition.next
-        self.inversion = definition.inversion
+        self.inversion = inversion
         
     def hasScaleDegree(self, note):
         return note in self.degrees
-    
+
+    @classmethod
+    def chordFromSymbol(cls, symbol, inversion):
+        fileName = symbol + "_info.json"
+        file = open(fileName)
+        definition = json.load(file)
+        return cls(definition, inversion)
+        
     def getSuccessors(self, note):
         successors = list()
-        for c in self.next:
-            if c.hasScaleDegree(note):
-                list.append(c)
+        for d in self.next: #d is a dictionary
+            for c in d[self.inversion]: #c is a number
+                candidate = Chord.chordFromSymbol(d["symbol"], c)
```

Comment:
I like this API, and this is fine for now, but I think there may be some issues down the line. Just so we're both thinking about it:

In this implementation, each time you check a next chord, you create a new chord instance and this could be a memory leak at scale. The Chord class is designed to be minimal so it can be slotted into different scenarios. What may be better is to have a global (or at least to a particular progression) chord registry. If a V chord as already been created, not need to recreate it if all we're doing is changing the inversion, melodic note, etc. Rather, there should be setter functions so we can update a chord's state. This could present some problems if we get into multithreading though, so we'll have to think on it. You can commit this for now.